lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1992,11 +1994,18 @@ def setUp(self):
         reload_django_url_config()
         super(TestRenderXBlock, self).setUp()
 
-    def get_response(self, url_encoded_params=None):
+    def test_render_xblock(self):
+        response = self.get_response(usage_key_string='jquery.js')
```

Comment:
better for readablity if we use `'some_invalid_usage_key'` instead of `'jquery.js'.`